selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/workspace/AbstractTestWorkspaceProvider.java

```diff
@@ -37,25 +33,25 @@
  * @author Anatolii Bazko
  */
 @Singleton
```

Comment:
You shouldn't apply `@Singleton` annotations to abstract types. It doesn't make sense.
Use it in the extending class so that it will work.